pkg/development/wasm/operations_test.go

```diff
@@ -1097,6 +1124,15 @@ assertFalse:
 						errors = append(errors, response.GetOperationsResults().Results[1].GetValidationResult().ValidationErrors...)
 					}
 				}
+
+				if tc.expectCheckTraces {
+					require.NotNil(t, errors[0].CheckDebugInformation)
+					require.NotNil(t, errors[0].CheckResolvedDebugInformation)
+
+					errors[0].CheckDebugInformation = nil
```

Comment:
add a comment clarifying this is unset so we don't have to craft the expected values in the test input.
Still, would it be much work to check the actual values?